plugins/modules/zos_job_submit.py

```diff
@@ -179,6 +179,10 @@
       description: The total lapsed time the JCL ran for.
       type: int
       sample: 0
+    execution_time:
+      description: The time the JCL executed for.
```

Comment:
This time I'm not adding the case that the job is still running, because in that case, zos_job_submit will "fail" unless a deploy and forget option was used using `wait_time_s=0`.

```suggestion
      description: Total duration time of the job execution, if it has finished.
```